include/toolkitAPI.h

```diff
@@ -569,6 +569,14 @@ int DLLEXPORT swmm_getLinkStats(int index, SM_LinkStats *linkStats);
  pre-allocated by the caller.
  @return Error code
 */
+int DLLEXPORT swmm_getGagePrecip(int index, double *rainfall, double *snowfall, double *total)
```

Comment:
Doxygen requires the doc string be before the function declaration, Please move the doc string

Reply:
@abhiramm7, also when you declare these functions you need to terminate the line with `;`

Reply:
@abhiramm7, the Appveyor CI is failing because of the last comment I made.  I think building the project locally before committing/pushing will really help identify errors :-)

Reply:
Would me being on a mac be a issue ? @bemcdonnell

Reply:
@abhiramm7, i doubt it.  Just to reiterate, your two new function declarations inside toolkitAPI.h are missing `;` at the end.

Reply:
@bemcdonnell Updated docstrings